packages/expo-updates/ios/EXUpdates/UpdatesUtils.swift

```diff
@@ -96,6 +94,26 @@ public final class UpdatesUtils: NSObject {
     return config.runtimeVersion ?? config.sdkVersion ?? "1"
   }
 
+  internal static func embeddedAssetsMap(withConfig config: UpdatesConfig, database: UpdatesDatabase?) -> [String: Any] {
```

Comment:
```suggestion
  internal static func embeddedAssetsMap(withConfig config: UpdatesConfig, database: UpdatesDatabase) -> [String: String?] {
```

Reply:
The return type was chosen to match the type of the `assetFilesMap` property in `AppLauncherWithDatabase`.

Reply:
We should generally return as constrained a type as possible. If we return any, one could theoretically refactor this and accidentally return a different value type, implicitly breaking the code. Having the more constrained type gives us one more layer of safety during refactors.

In a follow-up we should consider constraining the type of `assetFilesMap` as well.

Reply:
... and this property is defined in the `AppLauncher` protocol. I'll check to see how to safely make this change.

Reply:
Yeah, feel free to do it in a follow-up. Definitely want to avoid using `Any` in new code as much as possible though as it generally indicates a weakness in our structure.